Core/Utilities/TypedList.py

```diff
@@ -1,5 +1,6 @@
+"""  UNUSED UNUSED UNUSED UNUSED UNUSED
```

Comment:
remove it ?

Reply:
https://github.com/DIRACGrid/DIRAC/pull/4796